### PR TITLE
Update dashboard install using latest `lsql` release

### PR DESCRIPTION
**Pull request by JCZuurmond**
### Linked issues
Dashboard issues arose after formatting the quries

### Functionality

- [x] modified existing command: `databricks labs instal ucx`

### Tests

- [x] manually tested
- [x] verified on staging environment (screenshot attached)

### CODE REVIEW

**Comment by JCZuurmond** [comment]
https://github.com/user-attachments/assets/4b2884e4-f1ce-4107-bc03-751628d83611


https://github.com/user-attachments/assets/75c4cc93-93aa-4c07-a27b-acc85d12b881


https://github.com/user-attachments/assets/89ae8370-0f5f-48f9-9df6-a6969d94dd1a

Screenshot 2024-07-25 at 16 00 10
Screenshot 2024-07-25 at 16 03 49

File: src/databricks/labs/ucx/queries/assessment/main/01_0_compatibility.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* --title 'UC readiness' --width 2 --height 4 */
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/queries/assessment/main/01_0_compatibility.sql:1]: These widgets were not spaced correctly anymore, see last screen recording below to see how it looks now 👇

File: src/databricks/labs/ucx/install.py [hunk 2]
```diff
@@ -562,14 +562,19 @@ def _create_dashboard(self, folder: Path, *, parent_path: str | None = None) ->
         )
         metadata.display_name = f"{self._name('UCX ')} {folder.parent.stem.title()} ({folder.stem.title()})"
         reference = f"{folder.parent.stem}_{folder.stem}".lower()
-        dashboard = Dashboards(self._ws).deploy_dashboard(
-            metadata.as_lakeview(),
-            dashboard_id=self._install_state.dashboards.get(reference),
+        dashboard_id = self._install_state.dashboards.get(reference)
+        if dashboard_id is not None and "-" in dashboard_id:  # Upgrading from Redash to Lakeview
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/install.py:566]: Need to confirm that this works. @nfx : Can you confirm that the Redash ids always contain a "-" and Lakeview does not

Comment (nfx) [reply src/databricks/labs/ucx/install.py:566]: integration tests will confirm :)

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**